tests/foreman/ui/test_oscapcontent.py

```diff
@@ -0,0 +1,95 @@
+from robottelo.constants import (
+    ANY_CONTEXT,
+    OSCAP_CONTENT_FILE,
+    OSCAP_DEFAULT_CONTENT,
+)
+from robottelo.datafactory import valid_data_list
+from robottelo.helpers import get_data_file
+from robottelo.test import UITestCase
+from robottelo.ui.factory import set_context, make_oscapcontent
+from robottelo.ui.session import Session
+
+
+class OpenScapContent(UITestCase):
+    """Implements Oscap Content tests in UI."""
+
+    @classmethod
+    def setUpClass(cls):
+        super(OpenScapContent, cls).setUpClass()
+        cls.content_path = get_data_file(OSCAP_CONTENT_FILE)
+
+    def test_positive_create_oscapcontent(self):
+        """@Test: Create OpenScap content.
+
+        @Feature: OpenScap - Positive Create.
+
+        @Steps:
+
+        1. Create an openscap content.
+        2. Provide all the appropriate parameters.
+
+        @Assert: Whether creating  content for OpenScap is successful.
+
```

Comment:
I have no issues dropping it, but currently this is how all the test modules are written or is it that we decided to change this from now on?